apps/namada-interface/src/slices/transfers.ts

```diff
@@ -235,9 +237,11 @@ const createTransfer = async (
   const txCode = await fetchWasmCode(TxWasm.Transfer);
 
   // Invoke extension integration
-  const anoma = new Anoma(chains[chainId]);
-  anoma.connect();
-  const signer = anoma.signer();
+  const integration = getIntegration(chainId);
+  integration.detect();
+  //TODO: We have to treat this as anoma Signer for now
```

Comment:
This is fine, as use with any other extension means we'll need to use IBC actions, which will be handled separately. Only Namada will use Signer this way